From 344654356babcfebcc05e77bff9bcd7097277dab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Mon, 20 Oct 2025 15:33:26 +0200
Subject: [PATCH] Wrap text even if words are too long

This ensures that the width of the text view never gets larger, even if
the text cannot be split by words.
---
 src/callback.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/callback.c b/src/callback.c
index 7bda2c6..c2d253c 100644
--- a/src/callback.c
+++ b/src/callback.c
@@ -295,7 +295,7 @@ void on_option_word_wrap(void)
 	state = gtk_toggle_action_get_active(
 		GTK_TOGGLE_ACTION(gtk_ui_manager_get_action(pub->mw->menubar, "/M/Options/WordWrap")));
 	gtk_text_view_set_wrap_mode(GTK_TEXT_VIEW(pub->mw->view),
-		state ? GTK_WRAP_WORD : GTK_WRAP_NONE);
+		state ? GTK_WRAP_WORD_CHAR : GTK_WRAP_NONE);
 }
 
 void on_option_line_numbers(void)
